tests/integration/Owncloud/OcisPhpSdk/OcisTest.php

```diff
@@ -89,6 +89,30 @@ public function testGetGroups(): void
             $this->assertIsArray($group->getMembers());
         }
     }
+
+    /**
+     * @return void
+     */
+    public function testAddUserAndGroupsExpand(): void
```

Comment:
what do you want to test here? adding user or getting group and expanding the members?
If you want to test both, write two separate tests